src/main/resources/contexts/resource-context.json

```diff
@@ -1,46 +1,16 @@
 {
   "@context": {
     "nxv": "https://bluebrain.github.io/nexus/vocabulary/",
-    "Schema": {
-      "@id": "nxv:Schema",
-      "@type": "@id"
-    },
-    "File": {
-      "@id": "nxv:File",
-      "@type": "@id"
-    },
-    "Resolver": {
-      "@id": "nxv:Resolver",
-      "@type": "@id"
-    },
-    "InProject": {
-      "@id": "nxv:InProject",
-      "@type": "@id"
-    },
-    "CrossProject": {
-      "@id": "nxv:CrossProject",
-      "@type": "@id"
-    },
-    "View": {
-      "@id": "nxv:View",
-      "@type": "@id"
-    },
-    "ElasticSearchView": {
-      "@id": "nxv:ElasticSearchView",
-      "@type": "@id"
-    },
-    "AggregateElasticSearchView": {
-      "@id": "nxv:AggregateElasticSearchView",
-      "@type": "@id"
-    },
-    "SparqlView": {
-      "@id": "nxv:SparqlView",
-      "@type": "@id"
-    },
-    "Alpha": {
-      "@id": "nxv:Alpha",
-      "@type": "@id"
-    },
+    "Schema": "nxv:Schema",
```

Comment:
why this change?

Reply:
Because it is not needed. Those aliases are for things like
```json
{
   "@type": "Schema"
}
```
So an alias is enough, as you can check with the playground.